.github/CODEOWNERS

```diff
@@ -1,116 +1,55 @@
 * @bader
 
 # Front-end compiler
-clang/ @premanandrao @elizabethandrews @smanna12
+clang/ @intel/dpcpp-cfe-reviewers
 
-# Driver
-clang/**/Driver @mdtoguchi @AGindinson @hchilama
+# Clang driver
+clang/**/Driver @intel/dpcpp-clang-driver-reviewers
```

Comment:
Please, make sure that all these teams exist and have write access to the repository.

Reply:
Good catch. Fixed for all teams both in intel/llvm and intel/llvm-test-suite.

Reply:
Okay. Please, test automatic assignment. I think it's still broken due to the issue in my previous comment.

Reply:
it worked on change of the sources in the PR (fixed 2 hours form now). No automatic assignemnt for existent PR.